src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(7 * $day-size + 6 * $gap + 2 * $padding);
+  padding: 5px;
+  margin: auto;
+  gap: $gap;
+
+  @each $day, $index in $days {
```

Comment:
Make sure that the `$days` variable is defined and contains the correct values for the days of the week. This is crucial for the `@each` loop to function correctly.